charts/s3gw/values.yaml

```diff
@@ -223,6 +226,7 @@ cosi:
     imageRegistry:
     imagePullPolicy:
     name:
+    endpoint:
```

Comment:
Besides development, is there ever a good reason to specify the endpoint manually?

Reply:
The main reason, is that the default values with the `http://` schema.
One could want to attach the driver to secure port with the `https://` schema. I haven't tested if this works for real, but the reasoning was mainly this.

Reply:
Ah yes of course, that makes sense - although if it works, I'd prefer the schema to automatically select HTTPS if there are certificates configured. But until the driver is confirmed to work on a TLS connection this solution will do just fine

Reply:
yup, since I'm currently working to setup a proper test framework for COSI we can defer the change when we will have a way to test this.